src/main/java/uk/gov/hmcts/reform/notificationservice/model/out/NotificationResponse.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.notificationservice.model.out;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import uk.gov.hmcts.reform.notificationservice.util.CustomInstantSerializer;
+
+import java.time.Instant;
+
+public class NotificationResponse {
+    @JsonProperty("notification_id")
```

Comment:
this is never tested. Perhaps mapping to this model could happen in controller layer instead of the service layer?
(and then we can verify what the resulting json is in tests for controller)

Reply:
Exactly, it is supposed to be tested along with the future controller class

Reply:
I'm suggesting returning existing `Notification` model from the service layer and add this API model along with the controller later on

Reply:
Have done